oauth2/oauth2-resource-server/src/test/java/org/springframework/security/oauth2/server/resource/web/server/ServerBearerTokenAuthenticationConverterTests.java

```diff
@@ -155,7 +155,8 @@ public void resolveWhenHeaderWithInvalidCharactersIsPresentAndNotSubscribedThenN
 	}
 
 	@Test
-	public void resolveWhenValidHeaderIsPresentTogetherWithQueryParameterThenAuthenticationExceptionIsThrown() {
+	public void resolveWhenValidHeaderIsPresentTogetherWithQueryParameterThenAuthenticationExceptionIsThrown1() {
```

Comment:
If this test is not overlapping with other tests, is it possible to change the name? In this test, perhaps also mention `AndQueryParameterSupported` as above?